app/services/atmosphere/cloud/vm_creator.rb

```diff
@@ -27,7 +20,27 @@ def execute
         server_params[:nics] = [{ net_id: @nic }]
       end
 
-      set_security_groups!(server_params)
+      case(@tmpl.compute_site.technology)
+        when 'azure'
+          server_params[:vm_name] = SecureRandom.uuid
+          server_params[:vm_user] = 'atmosphere'
+          server_params[:password] = 'TomekiPiotrek2015' # TODO: Replace with key injection
+          server_params[:public_key_path] = '/home/tomek/.ssh/tomek.pub'
+          server_params[:image] = tmpl_id
+          server_params[:location] = "North Central US"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.